### DISCUSSION TITLE
Display a notification after a `bk-dynamic-form-drawer` http action

### DISCUSSION THREAD

**Discussion by davidgarra** [mia-platform/community#355] [Q&A] [answered]
### Which section?

Backoffice

### Which is the version?

back-kit@1.4.7, microlc-backend@0.8.1, microlc-frontend@0.9.0

### Description

I've created a `bk-dynamic-form-drawer` with an `action` of type `http`. I am trying to display a notification once the action is executed, but I'm struggling to find a solution.

I've added the `triggeredBy` property to the action in order to catch them in the `bk-notifications` component, but it doesn't seem to function as expected...

This is the code:
```json
  {
    "type": "element",
    "tag": "bk-dynamic-form-drawer",
    "properties": {
      "dataSchema": {
        "$ref": "dataSchema"
      },
      "requireConfirm": true,
      "actions": {
        "select": [
          {
            "closeAfter": true,
            "content": "Add Followup",
            "action": {
              "type": "http",
              "config": {
                "url": "/followup-endpoint",
                "method": "POST",
                "body": {
                  "orderId": "{{values.orderId}}",
                  "date": "{{rawObject values.followupDate}}"
                },
                "triggeredBy": "create-followup"
              }
            }
          }
        ]
      }
    }
  },
  {
    "type": "element",
    "tag": "bk-notifications",
    "properties": {
      "rootElementSelectors": "#microlc-element-composer",
      "successEventMap": {
        "create-followup": {
          "title": {
            "it": "Creazione follow-up",
            "en": "Follow-up creation"
          },
          "content": {
            "it": "Follow-up creato correttamente.",
            "en": "Follow-up successfully created"
          },
          "type": "success"
        }
      },
      "errorEventMap": {
        "create-followup": {
          "title": {
            "it": "Creazione follow-up",
            "en": "Follow-up creation"
          },
          "content": {
            "it": "Errore nella creazione del follow-up.",
            "en": "Error creating follow-up"
          },
          "type": "error"
        }
      }
    }
  }
```

Can anyone help me?

### Actual Outcome

The action is executed correctly, but no notification banner appears, neither in the case of a 204 response, nor in the case of a 400 response.

### Expected Outcome

The notification banner should appear at the end of the http call.

**Comment by TheRusCompass** [comment]
Hi @davidgarra! Thanks for reaching us! We are going to release a fix about it in the next few days, we will let you know as soon as it is ready with all the required details.

**Reply by davidgarra** [reply]
perfect, I'm looking forward to the solution!!!

**Reply by TheRusCompass** [reply, marked as answer]
Hello, the fix for this problem has been released with the back-kit library version 1.4.10. You can have a look at the [changelog here](https://docs.mia-platform.eu/docs/microfrontend-composer/back-kit/changelog). Anyway, using this new version, the `bk-dynamic-form-drawer` should be able to correctly trigger the actions.

**Reply by davidgarra** [reply]
great, thanks a lot!!!
